libavfilter/vf_dnn_detect.c

```diff
@@ -249,7 +332,17 @@ static av_cold int dnn_detect_init(AVFilterContext *context)
     ret = ff_dnn_init(&ctx->dnnctx, DFT_ANALYTICS_DETECT, context);
     if (ret < 0)
         return ret;
-    ff_dnn_set_detect_post_proc(&ctx->dnnctx, dnn_detect_post_proc);
```

Comment:
still use dnn_detect_post_proc as a single entry, but to go to different path within this function.